packages/metrics-cordova/src/AppMetrics.ts

```diff
@@ -0,0 +1,23 @@
+import {Metrics} from '@aerogearservices/metrics';
+
+declare var device: any;
+
+/**
+ * Collect application metrics:
+ *
+ *  - appId - application bundle id
+ *  - appVersion - version of the application
+ *  - sdkVersion - AeroGear Services SDK version
+ */
+export class AppMetrics implements Metrics {
```

Comment:
is there any reason this is cordova specific? The only cordova specific thing to me is the platform. Other values can be used cross-platform?

Reply:
App version will be as well.

Reply:
Just experimenting with different solutions so code may be little bit raw in some cases.